galaxy/values.yaml

```diff
@@ -79,7 +79,7 @@ galaxy_backend_postgres: false
 
 legacy:
    pre_k8s_16: false
-rbac_needed: false
```

Comment:
My concerns on this one is that minikube with Kubernetes below 1.10 seems to have rbac disabled by default, where as minikube with k8s >= 1.10 has it enabled. Trying to start this on minikube k8s < 1.10 with `rbac_needed` will produce an error I think. Since k8s 1.9 is still widely in use, I would leave the default behaviour as it is.

Reply:
That sounds reasonable.  I'll change that.  Let's remember to document the fact that `rbac_needed` must be set to true for more recent versions of minikube.